src/test/java/oop/beverages/BeveragesTest.Java

```diff
@@ -0,0 +1,39 @@
+package oop.beverages;
+
+/**
+ * New class that defines all Lattes from Coffee
+ */
+public final class Latte extends Coffee{
+    
+    /**
+     * Contructor method to get flavor for Latte
+     * @param Flavor
+     */
+    public Latte(String Flavor){
+        this.setFlavor(Flavor);
+    }
+
+    /**
+     * PlaceOrder() makes a new Latte and places it into the order queue
+     * @param Flavor
+     */
+    public void PlaceOrder(String Flavor){
+        Latte OrderNumber = new Latte (Flavor);
+        this.getOrder().add(OrderNumber);
+    }
+
+    /**
+     * TakeOrder() removes first Latte from order queue and returns the order number
+     */
+    public Latte TakeOrder() throws Exception{
```

Comment:
watch style it should be takeOrder().

I'd also add - intuitively a takeOrder() method would accept an instance of Latte and add it to the orders list. this method implementation reads more like a "serveOrder()" to me personally.